integration/kubernetes/init.sh

```diff
@@ -274,6 +274,10 @@ start_cri_runtime_service() {
 
 	sudo systemctl status "${cri}" --no-pager || \
 		die "Unable to start the ${cri} service"
+	if [ ! -f /opt/cni/bin/flannel ]; then
```

Comment:
Hi @BbolroC, if the issue only occurs in s390x, so is it better to add an $ARCH check and run this logic only for s390x?

Reply:
@fidencio told me it happened during the TDX CI tests as well. I think it is common across arches.

Reply:
I think we can restrict it to bear metal test environment instead of just s390x.